lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -557,23 +607,61 @@ def addMatSection():
                     if hasProcessedMaterial:
                         continue
                     # Shader attributes are special
-                    self.createShaderAttributesSection()
+                    schemasAttributes['shader'] = []
                     hasProcessedMaterial = True
                     # Note: don't show the material section for materials.
                     self.addedMaterialSection = True
                 # We have a special case when building the Xformable section.
                 elif schemaTypeName == 'UsdGeomXformable':
-                    self.createTransformAttributesSection(sectionName, attrsToAdd)
+                    schemasAttributes['transforms'] = attrsToAdd
                 elif schemaTypeName == 'UsdGeomImageable':
-                    self.createDisplaySection(sectionName, attrsToAdd)
-                else:
-                    sectionsToCollapse = ['Curves', 'Point Based', 'Geometric Prim', 'Boundable',
-                                          'Imageable', 'Field Asset', 'Light']
-                    collapse = sectionName in sectionsToCollapse
-                    self.createSection(sectionName, attrsToAdd, collapse)
+                    schemasAttributes['display'] = attrsToAdd
+
+        return schemasAttributes
+
+    def createSchemasSections(self, schemasOrder, schemasAttributes):
```

Comment:
This is where the UI are created. There is map from schema names or special names to the "create" function used to handle the creation. Most go to the default createSection function, only special sections that need special code have their own function, like before.